mmcv/runner/hooks/optimizer.py

```diff
@@ -20,9 +21,27 @@
 
 @HOOKS.register_module()
 class OptimizerHook(Hook):
+    """A hook contains custom operations for the optimizer.
+
+    Args:
+        grad_clip (dict): A config dict to control the clip_grad.
+            Default: None.
```

Comment:
(dict) -> (dict, optional)
when the default value is `None`,  the type of keyword argument can be `optional`.